ama-sample.conf

```diff
@@ -1,17 +1,24 @@
 {
+  #required
   pg => 'postgresql://mojo:mojo@/test1',
+  #recommended
   secrets => ['mojo'],
+  #leave blank to remove login functionality
   oauth2 => {
+    #leave blank to remove facebook login
     facebook => {
             key => 'referred to as app ID',
             secret => 'referred to as app-secret',
     },
+    #leave blank to remove google login
     google => {
             key => 'referred to as client ID',
             secret => 'referred to as client secret'
     },
   },
-  vote_floor => -2,
+  #required -2 recommended
+  vote_floor => -int,
+  #leave blank to remove email feedback functionality
```

Comment:
A link to how to set up a Sendgrid API and to docs on what values (e.g. to / from) are allowed here.